src/routes/Edit/VideoSplitter.tsx

```diff
@@ -185,6 +210,17 @@ const VideoSplitter = ({
   };
 
   const saveChanges = async () => {
+    console.log(currentSegment?.title.length);
+    if (!currentSegment?.title.length || !currentSegment?.description.length) {
```

Comment:
This is nice and definitely needed. If possible it would be nice to integrate a more formal form validation process, however this is perfectly fine for now.

Reply:
We will do that. Issue created.